### DISCUSSION TITLE
MDTraj 1.10.1

### DISCUSSION THREAD

**Discussion by mattwthompson** [mdtraj/mdtraj#1942] [Announcements]
v1.10.1 (November 4, 2024)
--------------------------

- Update RMSD module for Cython 3.0 (#1917)
- Fix edge case with float rounding in angle computation (#1841)
- Add atom selection via atom_indices to SASA computation (#1844)
- ReadTheDocs migration (#1888)
- Fix netCDF import (#1894)
- Use standard library `ast.unparse` instead of third-party `astunparse` (#1898)
- Allow appending to HDF5 files (#1899)
- Remove deprecated `oldest-supported-numpy` (#1902)
- Remove remaining uses of `six` (#1908)
- Replace `xdrlib` with `mda_xdrlib` (#1911)
- Add `mda-xdrlib` to `setup.py` (#1913)
- Set up `versioneer` (#1909)
- Remove `distutils` from tests (#1916)
- Remove unused `xdrlib` import from Python layer (#1919)
- Fix PDB "vmd convention" false-positive (#1904)

A total of 11 people contributed to this release.
People with a "+" by their names contributed a patch for the first time.

- Charlie Laughton
- Guillermo Pérez-Hernández
- Sukrit Singh
- Jeremy Leung
- Sander Roet
- Chris Jones+
- David W.H. Swenson
- Matthew W. Thompson
- Robert Schütz+
- Patrick Kunzmann
- Augustin Zidek+

<hr /><em>This discussion was created from the release <a href='https://github.com/mdtraj/mdtraj/releases/tag/1.10.1'>MDTraj 1.10.1</a>.</em>

**Comment by jeremyleung521** [comment]
source + wheels for `macos_arm64` / `macos_x86_64` / `linux_x86_64` python 3.10-3.12 should now be up on pypi. 

I might eventually build wheels for linux_arm64.

**Reply by jeremyleung521** [reply]
Just uploaded the aarch64 linux wheels as well, in case someone needs them.

**Comment by mattwthompson** [comment]
`conda-forge` builds are up within the hour
